wp-content/plugins/core/src/Templates/Controllers/Traits/Copyright.php

```diff
@@ -4,7 +4,7 @@
 namespace Tribe\Project\Templates\Controllers\Traits;
 
 trait Copyright {
-	protected function get_copyright() {
+	public function copyright(): string {
 		return sprintf( __( '%s %d All Rights Reserved.', 'tribe' ), '&copy;', date( 'Y' ) );
```

Comment:
Also used in the unsupported browser template. I wanted a single place to update it.

Reply:
Also consider a project where the copyright statement is coming from a setting. This will allow us to change the logic to grab the setting in a single location.